frontend/src/components/Modal/RiderModalInfo.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import cn from 'classnames';
+import { Button, Input, Label } from '../FormElements/FormElements';
+import styles from './ridermodal.module.css';
+import { ObjectType } from '../../types/index';
+
+type ModalFormProps = {
+  onSubmit: (data: ObjectType) => void;
+  formData?: ObjectType;
+}
+
+const RiderModalInfo = ({ onSubmit }: ModalFormProps) => {
+  const { register, handleSubmit } = useForm();
+  const beforeSubmit = ({ name, netid, email, phoneNumber, needs,
+    address, start, end }: ObjectType) => {
+    const startDate = new Date(`${start}`).toISOString();
+    const endDate = new Date(`${end}`).toISOString();
+    const splitName = name.split(' ');
+    const firstName = splitName[0];
+    const lastName = splitName[1];
+    const accessibilityNeeds = needs.split(',');
+    onSubmit({
+      id: netid,
+      firstName,
+      lastName,
+      email,
+      phoneNumber,
+      accessibilityNeeds,
+      address,
+      startDate,
```

Comment:
What does a rider's "startDate" and "endDate" mean?

Reply:
^same as above